basic_portfolio/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Basic Portfolio Page</title>
+<style>
+h1{
```

Comment:
So there are headline elements, which are h1 - h7, but there are also semantic elements called "header" elements.  Basically they act like divs, but give some clue as to what the purpose of them is.  Check them out <a href="http://www.w3schools.com/html/html5_semantic_elements.asp" target="_blank">here </a>